dist-material/release-docs/NOTICE

```diff
@@ -914,4 +914,15 @@ MessagePackage Notice
 This product includes the software developed by third-party:
 
  * Google Guava https://code.google.com/p/guava-libraries/  (APL2)
- * sbt-extras: https://github.com/paulp/sbt-extras  (BSD)  (LICENSE.sbt-extras.txt)
\ No newline at end of file
+ * sbt-extras: https://github.com/paulp/sbt-extras  (BSD)  (LICENSE.sbt-extras.txt)
+
+========================================================================
```

Comment:
This is standard Notice of ASF. As we are both belong to the foundation. We don't need this

Reply:
dropped